examples/common/customize.env

```diff
@@ -45,3 +45,6 @@
 # Configuration for Conjur authn-k8s
 #export ANNOTATION_BASED_AUTHN="<true-or-false-defaults-to-true>"
 #export AUTHENTICATOR_ID="<authenticator-id-string-defaults-to-my-authenticator-id>"
+
+# Default authn strategy is authn-k8s. But we want to support ad of authn-jwt
```

Comment:
Can you clarify the last part of this comment? I'm not 100% sure what it's trying to say